warehouse/query-core/src/main/java/datawave/query/attributes/Attribute.java

```diff
@@ -155,64 +155,48 @@ protected void clearMetadata() {
     }
     
     protected void writeMetadata(DataOutput out, Boolean reducedResponse) throws IOException {
-        out.writeBoolean(reducedResponse);
-        
-        if (!reducedResponse) {
-            out.writeBoolean(isMetadataSet());
-            if (isMetadataSet()) {
-                byte[] cvBytes = getColumnVisibility().getExpression();
-                
-                WritableUtils.writeVInt(out, cvBytes.length);
-                
-                out.write(cvBytes);
-                out.writeLong(getTimestamp());
-            }
+        out.writeBoolean(isMetadataSet());
```

Comment:
This is no longer based on the reducedResponse boolean being sent in.  Is that what you meant to do?  I think the metadata will basically always be set on the attribute.
The reduced response was an effort to reduce the amount of data being sent back to the client (webserver/microservice).  If you are removing that concept altogether, then we shouldn't have a reduced response boolean here nor should that option be sent from the DefaultQueryPlanner over to the QueryIterator. via the scan options.

Reply:
Yes, that is what I meant to do. I can work on ripping it all the way out. I was initially targeting a quick bug fix to get the events marked correctly. I'll start working on fully taking it out though.

Reply:
Had a discussion with Ivan. We're going to merge this as-is and I opened up a follow on ticket for a better long-term fix, which will be serializing the document level colvis always and then reverting this change.